SwanContents/swancontents/filemanager/eos/fileio.py

```diff
@@ -107,12 +108,11 @@ def _get_os_path(self, path):
         else:
             return super()._get_os_path(path)
 
-    def _save_notebook(self, os_path, nb, capture_validation_error=None):
+    async def _save_notebook(self, os_path, nb, capture_validation_error=None):
         """
         Save a notebook on EOS via FUSE with the default routine.
         Plus, store a copy of the same notebook on the host machine via docker volume.
         """
-        import time
```

Comment:
Won't this removal cause a lookup error in line 138?

Reply:
Ah I missed it, but the method has been fully removed. The only part that was kept was the one that calls the atomic_writing context manager.